beacon-chain/p2p/service.go

```diff
@@ -30,7 +30,8 @@ import (
 
 var _ = shared.Service(&Service{})
 
-var pollingPeriod = 1 * time.Second
+// check local table every 30 seconds for newly added peers.
+var pollingPeriod = 30 * time.Second
```

Comment:
is this formal enough to be defined in beacon's `config.go`?

Reply:
Not really, most of these params are only useful in the `p2p` package. This is just a baseline on how often to poll our local table for new peers. I think they are fine living here in this package.